### DISCUSSION TITLE
Could not render the variants of  enum

### DISCUSSION THREAD

**Discussion by ff-anthony-soto** [prisma/prisma#2140] [Q&A] [answered]
I was trying to migrate my schema with the new changes added in the last release.

But I got an error after executing `npx prisma2 migrate save --experimental` .

First try:

<img width="1280" alt="Screen Shot 2020-04-08 at 3 38 50 PM" src="https://user-images.githubusercontent.com/59031228/78836649-c89a1400-79af-11ea-8c12-cc02a7d14df9.png">

Second try:

<img width="1280" alt="Screen Shot 2020-04-08 at 3 21 33 PM" src="https://user-images.githubusercontent.com/59031228/78836641-c5068d00-79af-11ea-9bc5-a74780f48ab8.png">

Models associates:

```
model User {
  id    String @id @default(uuid())
  personId Int @map(name: "person_id")
  userTypeId Int @map(name: "user_type_id")
  email String @unique
  avatar String
  status UserStatus
  
  person Person @relation(fields: [personId], references: [id])
  type UserType @relation(fields: [userTypeId], references: [id])

  createdAt DateTime @default(now()) @map(name: "created_at")
  updatedAt DateTime @default(now()) @map(name: "updated_at")

  @@map(name: "users")
}


enum UserStatus {
  CONFIRMED
  CANCELED
  BLOCKED
}
```

**Comment by janpio** [comment, marked as answer]
Please open an issue over at https://github.com/prisma/migrate/issues/ 👍

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
